src/blog/_posts/2016-03-15-opera-36.md

```diff
@@ -75,32 +75,42 @@ Two new global events, `unhandledrejection` and `rejectionhandled`, can be used
 
 [The CSS Variables spec](https://drafts.csswg.org/css-variables/) defines a new primitive value type that is accepted by all CSS properties, as well as custom properties for defining them. As with other CSS values, they can be updated programmatically using JavaScript. See [“CSS Variables: why should you care?”](https://developers.google.com/web/updates/2016/02/css-variables-why-should-you-care) for more info. [A demo is available.](https://googlechrome.github.io/samples/css-custom-properties/)
 
-## Case-insensitive attribute name and value selectors
+## Case-insensitive attribute value selectors
 
-[In HTML, element and attribute names are ASCII case-insensitive](https://html.spec.whatwg.org/multipage/scripting.html#case-sensitivity), but in XML, they are case-sensitive.
+In HTML, the attribute *values* of [certain attributes are compared ASCII-case-insensitively](https://html.spec.whatwg.org/multipage/scripting.html#case-sensitivity).
 
 Consider the following CSS:
 
-	/* Every paragraph whose ID starts with `note` gets a lime background. */
+	/* Every paragraph whose ID starts with `note` (case-sensitively) gets a lime background. */
 	p[id^="note"] {
-		color: lime;
+		background-color: lime;
+	}
+	/* Every paragraph whose `lang` is `no` (ASCII-case-insensitively, in HTML) get underlined */
+	p[lang="no"] {
+		text-decoration: underline;
```

Comment:
I don’t really like this example as I prefer `:lang(foo)` over `[lang="foo"]`, but hey. (Should I stop doing that?)